app/src/main/res/values/colors.xml

```diff
@@ -28,6 +28,7 @@
     <color name="snackbar_action_text">#3bdfff</color>
 
     <color name="black_38">#9E9E9E</color>
+    <color name="black_87">#de000000</color>
```

Comment:
This color is used from `com.github.ParkSangGwon:TedPicker` library. So extracted it `colors.xml` and removed this dependecy